src/styles/normalise.css

```diff
@@ -0,0 +1,24 @@
+* {
```

Comment:
typo in filename normalise => normalize

Reply:
done

Reply:
can be both)
<img width="687" alt="image" src="https://user-images.githubusercontent.com/53611910/202933440-ef26708a-5c3b-4703-aee4-bad96fd31c4e.png">

Reply:
both are correct)
<img width="698" alt="image" src="https://user-images.githubusercontent.com/53611910/202933632-e9932f0e-c4f5-4117-89b1-35c6c34bdaf5.png">